apcd-cms/Makefile

```diff
@@ -8,7 +8,7 @@ DOCKER_IMAGE_BRANCH := $(DOCKERHUB_REPO):$(shell git describe --exact-match --ta
 
 #.PHONY: build
 build:
-	docker-compose -f docker-compose.dev.yml build
+	docker compose -f docker-compose.dev.yml build
```

Comment:
Replace with https://github.com/TACC/Core-CMS-Custom/blob/main/apcd_cms/Makefile. Otherwise it will hit merge conflicts

Reply:
Done!